PlayerControls/sources/ContentControlsViewController.swift

```diff
@@ -121,6 +121,14 @@ extension ContentControlsViewController.Props.Player.Item {
             public init() { }
         }
         
+        public var pictureInPicture: PictureInPicture = .unsupported
+        public enum PictureInPicture {
+            case unsupported
+            case supported(toggle: Action<Void>, state: PipState)
```

Comment:
Ok, we should have it for `inactive` , but for `active` we don't need it - it is placed on the native PiP UI.